packages/runtime/src/templates/getHandler.ts

```diff
@@ -177,15 +190,25 @@ const makeHandler = (conf: NextConfig, app, pageRoot, staticManifest: Array<[str
   }
 }
 
-export const getHandler = ({ isODB = false, publishDir = '../../../.next', appDir = '../../..' }): string =>
+export const getHandler = ({
+  isODB = false,
+  publishDir = '../../../.next',
+  appDir = '../../..',
+  nextServerModuleRelativeLocation,
+}): string =>
   // This is a string, but if you have the right editor plugin it should format as js
   javascript/* javascript */ `
+  if (!${JSON.stringify(nextServerModuleRelativeLocation)}) {
+    throw new Error('Could not find Next.js server')
```

Comment:
In current state of this PR, this is thrown on lambda invocation and not during the build, so it restores the that behavior prior to https://github.com/netlify/next-runtime/pull/1950 on when the error is thrown - not the hill I will die on and I can move it back to build time.

The other changes in this PR make it more likely to find `next-server` than before, but I don't have 100% confidence in it, so that's why I move this error throwing back here as that was behavior for a long time.

Without that move, builds for fully static sites that hit current error are just blocked until we implement logic to determine wheter we need page route handlers at all and skip it altogether if not needed

Reply:
Yeah, i think that makes sense tbh. We only need the server at runtime so let's go with this.